parley/src/layout/mod.rs

```diff
@@ -96,21 +96,10 @@ impl<B: Brush> Layout<B> {
         self.data.full_width
     }
 
-    /// Returns the lower and upper bounds on the width of the layout.
-    pub fn content_widths(&self) -> ContentWidths {
-        self.data.content_widths()
-    }
-
-    /// Returns the minimum content width of the layout. This is the width of the layout if _all_
-    /// soft line-breaking opportunities are taken.
-    pub fn min_content_width(&self) -> f32 {
-        self.data.content_widths().min
-    }
-
-    /// Returns the maximum content width of the layout. This is the width of the layout if _no_
-    /// soft line-breaking opportunities are taken.
-    pub fn max_content_width(&self) -> f32 {
-        self.data.content_widths().max
+    /// Calculates the lower and upper bounds on the width of the layout. These
+    /// are recalculated every time this method is called.
```

Comment:
Aside: Should we mention the TODO about mixed direction text here?

This needn't happen/be resolved in this PR